java/maven/nbproject/project.properties

```diff
@@ -17,7 +17,6 @@
 
 javac.source=1.8
```

Comment:
modules which use nb-javac can't change their source level at the moment since they prepend the bootclasspath and this is only allowed on java 8.
Since target isn't (and can't) be set either once boot cp is modified, no API checks are active. Going to file an issue with more details.